microsoft/testsuites/vm_extensions/azure_keyvault_vm_extension.py

```diff
@@ -101,7 +101,10 @@ def verify_key_vault_extension(
         resource_group_name = runbook.shared_resource_group_name
         application_id = runbook.service_principal_client_id
         node_context = get_node_context(node)
-        vault_name = f"lisa-kve-{platform.subscription_id[-5:]}-{node_context.location}"
+        # A vault's name must be between 3-24 alphanumeric characters.
```

Comment:
Do you see it's failed some locations?

Reply:
Please follow below example to get last chars of locations. I reviewed last 11 chars last time, it has no conflict. It doesn't need to keep the full location name. The prefix need at least one extra char to know it's from kve, like "list-kv{sub id[:6]}{location[:11]}"

```python
def get_storage_account_name(
    subscription_id: str, location: str, type_: str = "s"
) -> str:
    subscription_id_postfix = subscription_id[-8:]
    # name should be shorter than 24 character
    return f"lisa{type_}{location[:11]}{subscription_id_postfix}"
```